sdk/identity/identity/src/msal/utils.ts

```diff
@@ -77,24 +77,28 @@ export function getKnownAuthorities(tenantId: string, authorityHost: string): st
  * @param logger - The logger of the credential.
  * @internal
  */
-export const defaultLoggerCallback: (logger: CredentialLogger) => msalCommon.ILoggerCallback = (
-  logger: CredentialLogger
+export const defaultLoggerCallback: (
+  logger: CredentialLogger,
+  platform?: "Node" | "Browser"
+) => msalCommon.ILoggerCallback = (
+  logger: CredentialLogger,
+  platform: "Node" | "Browser" = "Node"
```

Comment:
Could we compute this from `isNode/isBrowser` instead of making it an optional parameter? Or at least, make the default use isNode/isBrowser instead of just defaulting to Node.

Reply:
can do!